Dshop/apps/products_catalogue/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+
+class CategoryBaseModel(models.Model):
+    name = models.CharField(max_length=200, unique=True)
+    slug = models.SlugField(max_length=200, unique=True)
+    is_active = models.BooleanField(default=True)
+    sort = models.IntegerField(default=99)
+
+    class Meta:
+        ordering = ('sort',)
+        abstract = True
+
+    def __str__(self):
+        return self.name
+
+
+class Category(CategoryBaseModel):
+
+    class Meta:
+        verbose_name = 'Category'
+        verbose_name_plural = 'Categories'
+
+
+class SubCategory(CategoryBaseModel):
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+
+    class Meta:
+        verbose_name = 'Subcategory'
+        verbose_name_plural = 'Subcategories'
+
```

Comment:
I'd say subcategory is category with parent

```suggestion
class Category(CategoryBaseModel):
    parent = parent = models.ForeignKey('self', null=True, on_delete=models.CASCADE)

    class Meta:
        verbose_name = 'Category'
        verbose_name_plural = 'Categories'

```
This way we have unlimited nested categories and no need for base model.

Reply:
Refactored as requested with blank=True